### DISCUSSION TITLE
species richness mean and standard deviation?

### DISCUSSION THREAD

**Discussion by marwa38** [vegandevs/vegan#589] [Q&A] [answered]
If the species richness is the total number of observed species in the samples, is it possible to calculate the mean and SD for each sample? how?
![image](https://github.com/vegandevs/vegan/assets/68851733/1b1396cf-8b38-461f-b475-a78cbd0f9ef0)

Thanks in advance

**Comment by gavinsimpson** [comment]
The mean and standard deviation of what? You mentioned "samples" when referring to species richness, so are you asking if you can compute the mean of the richnesses of the set of samples?

Or are you thinking about something else?

**Comment by marwa38** [comment]
I don't think it make any sense to calculate the mean and SD for each sample as in the table. I am walkthrough a workflow (the attached table screenshot is part of), they calculate the mean and SD for each sample.
![image](https://github.com/vegandevs/vegan/assets/68851733/de4b915a-ca69-43ee-9897-38f458a6c1ee)

**Comment by jarioksa** [comment]
This is hardly a vegan issue.

**Reply by marwa38** [reply]
> This is hardly a vegan issue.

I am not reporting an issue 
I was asking what if it is possible to calculate mean and SD from the species richness.

**Comment by jarioksa** [comment, marked as answer]
Yes, it is possible — if you have several sampling units. I have no idea about the sampling design, but obviously they do have several sampling units per ”sample” (like they call it). Could be subplots, traps for animals, repeated sampling etc. Not even knowing the paper this is very difficult to guess. Asking on a platform with wider readership could be more fruitful, as this is unrelated to vegan and not many people come here except with vegan issues.

**Reply by marwa38** [reply]
Thanks for your comment
This what J wanted to hear in the discussion section (this is not an issue section as I can see)
For me, I won't know if this is something relevant the spices richness (they used vegan) or the sampling units. You clarified and J won't know if you haven't responded. My question is resolved! I didn't ask about the paper etc because this is not a vegan discussion.
Cheers
